cmd/lakefs/cmd/run.go

```diff
@@ -398,6 +401,55 @@ func checkRepos(ctx context.Context, logger logging.Logger, authMetadataManager
 	}
 }
 
+func deleteExpiredAddressesJob(ctx context.Context, c *catalog.Catalog) {
+	s := gocron.NewScheduler(time.UTC)
+	_, err := s.Every(1).Hour().Do(func() {
+		deleteExpiredAddressTokens(ctx, c)
+	})
+	if err != nil {
+		logging.Default().WithError(err).Debug("failed execute delete expired addresses job")
+	}
+
+	s.StartBlocking()
+}
+
+func deleteExpiredAddressTokens(ctx context.Context, c *catalog.Catalog) {
+	const (
+		jobWorkers = 3
+		size       = 10
+	)
+	rChan := make(chan *catalog.Repository, size)
+	for i := 0; i < jobWorkers; i++ {
```

Comment:
Is there a specific reason why we want to do it concurrently?
As it seems, there's no real significance for the time it takes for this job to run

Reply:
https://github.com/treeverse/lakeFS/issues/4959